src/js/kits/model-kit-select/KitPreview.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const DEFAULT_THUMB = '/static/images/thumb.png';
+
+const KitPreview = ({ kit, htmlName, selected=false, onToggle=() => {} }) => {
+    const [thumbImg, setThumbImg] = useState(kit.box_image.small || DEFAULT_THUMB);
+    const [errored, setErrored] = useState(false);
+
+    const onThumbError = () => {
+        if (errored) return;
+        setThumbImg(DEFAULT_THUMB);
+        setErrored(true);
+    };
+
+    const onChange = (event) => {
+        const { checked } = event.target;
+        onToggle(kit, checked);
+    };
+
+    return (
+        <div className="col-xs-12 col-sm-4 col-md-3 col-xl-2 preview">
+            <input
+                type="checkbox"
+                name={htmlName}
+                defaultChecked={selected}
+                id={`__modelkit_${kit.id}`}
+                value={kit.id}
+                onChange={ onChange }
+            />
+            <label htmlFor={`__modelkit_${kit.id}`}
+                   title={kit.name} className="thumbnail">
+                <img src={thumbImg} className="img-responsive" onError={ onThumbError } />
+                <span className="h5">
+                    <strong>
+                        { kit.name }
+                        { kit.kit_number ? <small>{ kit.kit_number }</small> : null}
```

Comment:
nope, it won't. it's in a format that's brand-specific, and always as string as it can be a combination of letters and numbers, e.g.:
![image](https://user-images.githubusercontent.com/5518550/82259009-2f213300-995b-11ea-86d9-7a950266f667.png)